pom.xml

```diff
@@ -254,6 +255,16 @@
         <artifactId>httpasyncclient</artifactId>
         <version>4.1.4</version>
       </dependency>
+      <dependency>
```

Comment:
move these into httpclient5 pom as we ran into trouble sometimes controlling versions in parent. you can leave the property here or move it to httpclient5